src/main/java/org/prnhs/javaee/swim/core/entity/User.java

```diff
@@ -0,0 +1,71 @@
+package org.prnhs.javaee.swim.core.entity;
+
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity(name = "users")
+public class User {
+    
+    @Id
```

Comment:
specifies to jpa that this field is the primary id/key of the object in the table.